### PR TITLE
New Design Shelly (PR updated for v3)

### PR DESCRIPTION
**Pull request by Thrunic**
I've updated Shelly to work with the v3 develop branch instead of lab2.5. I'm placing this PR and closing the previous (lab 2.5). This should get rid of the merge conflict with config/software/designs.json that was in the lab2.5 version and includes changes in sites/shared/hooks/use-design.mjs that are needed in develop.

### CODE REVIEW

File: designs/shelly/src/back.mjs [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  points.neckCP1 = points.neckShoulderCorner.shift(
+    necklineAngleAtRaglan + 180,
+    necklineArcLength / 3
+  )
+  points.neckCP2 = points.cfNeck.shift(0, necklineArcLength / 3)
+
+  const backNecklineToRaglanAngle = raglanAngle - (necklineAngleAtRaglan + 180)
+  store.set('backNecklineToRaglanAngle', backNecklineToRaglanAngle)
+
+  const drawSide = () => {
```
Comment (joostdecock) [inline designs/shelly/src/back.mjs:49]: Avoid defining functions in your draft method as they will be re-created on each draft.
Instead, move them out of the function body.

File: designs/shelly/src/base.mjs [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+  const sideAngle = points.sideHem.angle(points.armholeCornerScooped)
+  const sideLength = points.sideHem.dist(points.armholeCornerScooped)
+  points.sideCp1 = points.sideHem
+    .shift(sideAngle, (1 / 3) * sideLength)
+    .shift(sideAngle - 90, options.sideShape * sideLength)
+  points.sideCp2 = points.armholeCornerScooped
+    .shift(sideAngle + 180, (1 / 3) * sideLength)
+    .shift(sideAngle - 90, options.sideShape * sideLength)
+
+  const drawSide = () => {
```
Comment (joostdecock) [inline designs/shelly/src/base.mjs:108]: Same remark: Avoid defining functions in functions

**Review by joostdecock — CHANGES_REQUESTED** [review]
Can you also at (at least the English) translation. See the `i18n` folder and `src/index.mjs` of any other design.

File: designs/shelly/src/base.mjs (hunk 2 above)
Comment (Thrunic) [reply designs/shelly/src/base.mjs:108]: Fixed

**Comment by Thrunic** [comment]
Converted to draft because I will be adding an optional hood to the design, and as such, it would be better to hold off on merging it in at this point.

**Comment by Thrunic** [comment]
Shelly has been updated to work with the current V3 build and is ready for review.

**Comment by Thrunic** [comment]
Shelly should be good now. I've marked it for review.

**Comment by Thrunic** [comment]
It's running in the lab now, but there are manually edited/generated files that should be autogenerated, so I'm not marking it as ready to push yet. I need help getting it configured correctly.

**Comment by Thrunic** [comment]
Shelly is up-to-date, is working, and is ready for review.

**Comment by joostdecock** [comment]
Superseded by #5129

**Pull request closed without merging**